compiler/ccgexprs.nim

```diff
@@ -1003,23 +1003,7 @@ proc genAndOr(p: BProc, e: PNode, d: var TLoc, m: TMagic) =
   # tmp = a
   # end:
   # a = tmp
-  when false:
-    #if isSimpleExpr(e) and p.module.compileToCpp:
-    var tmpA, tmpB: TLoc
-    #getTemp(p, e.typ, tmpA)
-    #getTemp(p, e.typ, tmpB)
-    initLocExprSingleUse(p, e.sons[1], tmpA)
-    initLocExprSingleUse(p, e.sons[2], tmpB)
-    tmpB.k = locExpr
-    if m == mOr:
-      tmpB.r = "((" & rdLoc(tmpA) & ")||(" & rdLoc(tmpB) & "))"
-    else:
-      tmpB.r = "((" & rdLoc(tmpA) & ")&&(" & rdLoc(tmpB) & "))"
-    if d.k == locNone:
-      d = tmpB
-    else:
-      genAssignment(p, d, tmpB, {})
-  else:
+  when true:
```

Comment:
Nitpick: you could do the same as in https://github.com/nim-lang/Nim/pull/11981/files#diff-c771980a90bdbd4401f414dee2b14915R32 — without `when true`, just unindent its block. I think it is a cleaner solution.

Reply:
yea I can. I decided against it because in such a big PR I wanted to focus on trivial code changes. In the code you reference I didn't do it yet.